### PR TITLE
DO-NOT-MERGE: CMA-8690 : omniauth salesforce developers branch for omniauth-salesforce gem

### PR DESCRIPTION
**Pull request by vijayanands**
@anurag - please review the code changes. This branch is not to be merged to master (similar to what we did for salesforce-partners

### CODE REVIEW

**Comment by anurag** [comment]
I think you have the wrong Github handle!

On Mon, Aug 17, 2015 at 1:17 PM, Vijayanand Sankar <notifications@github.com

**Comment by vijayanands** [comment]
@anurag - sorry. my bad!!

**Comment by vijayanands** [comment]
@AnuragRamdasan - please review the code changes. This branch is not to be merged to master (similar to what we did for salesforce-partners)

File: lib/omniauth/strategies/salesforcedevelopers.rb [hunk 1]
```diff
@@ -82,17 +82,17 @@ def raw_info
...
+    #   default_options[:client_options][:site] = 'https://test.salesforce.com'
+    # end
+    #
+    # class DatabaseDotCom < OmniAuth::Strategies::Salesforcedevelopers
+    #   default_options[:client_options][:site] = 'https://login.database.com'
+    # end
+    #
+    # class SalesforcePreRelease < OmniAuth::Strategies::Salesforcedevelopers
+    #   default_options[:client_options][:site] = 'https://prerellogin.pre.salesforce.com/'
+    # end
```
Comment (AnuragRamdasan) [inline lib/omniauth/strategies/salesforcedevelopers.rb]: can just delete if not needed

**Comment by AnuragRamdasan** [comment]
:+1:

**Comment by AnuragRamdasan** [comment]
can this just be closed?

**Comment by vijayanands** [comment]
@AnuragRamdasan  - yes this can be closed now. I will do so

**Pull request closed without merging**